testdata/src/key_naming_case/key_naming_case.go

```diff
@@ -0,0 +1,38 @@
+package key_naming_case
```

Comment:
I wonder if it would be worth having 1 set of tests for each supported case format? Partly just to ensure there's coverage of all behaviours. But also because the case checks come from another library that might accidentally introduce regressions in upgrades for the formats not covered here.